connector/avro/src/test/scala/org/apache/spark/sql/avro/AvroFunctionsSuite.scala

```diff
@@ -260,8 +260,13 @@ class AvroFunctionsSuite extends QueryTest with SharedSparkSession {
       |  ]
       |}
     """.stripMargin
-    val avroSchema = AvroOptions(Map("avroSchema" -> avroTypeStruct)).schema.get
-    val sparkSchema = SchemaConverters.toSqlType(avroSchema).dataType.asInstanceOf[StructType]
+    val options = Map("avroSchema" -> avroTypeStruct)
+    val avroOptions = AvroOptions(options)
+    val avroSchema = avroOptions.schema.get
+    val sparkSchema = SchemaConverters.
+      toSqlType(avroSchema, options).
+      dataType.
+      asInstanceOf[StructType]
```

Comment:
Code style: `.` should move to start. E.g. :

```scala
val sparkSchema = SchemaConverters
      .toSqlType(avroSchema, options)
      .dataType
      .asInstanceOf[StructType]
```

Reply:
Thanks for catching it. I don't know why it became like that.